ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -24,31 +24,18 @@ Control {
     }
 
     property alias quickActions: quickActionsPanel.items
-    property alias statusChatInput: editComponent.inputComponent
+    property var statusChatInput
```

Comment:
It could (and in fact it is at the moment :) Matter of taste/preference I'd say; `var` will work for either `Component` or anything else and that way we might change the implementation without breaking the public API